packages/server/lib/plugins/child/run_plugins.js

```diff
@@ -226,6 +227,8 @@ module.exports = (ipc, pluginsFile, projectRoot) => {
   }
 
   ipc.on('load', (config) => {
+    config.version = pkg.version
```

Comment:
Instead of adding version to the config here, let's add it [in the same place we add the projectRoot and configFile](https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/plugins/index.js#L87).